components/hal/esp32c3/include/hal/clk_gate_ll.h

```diff
@@ -40,8 +40,8 @@ static inline uint32_t periph_ll_get_clk_en_mask(periph_module_t periph)
         return SYSTEM_UART1_CLK_EN;
     case PERIPH_I2C0_MODULE:
         return SYSTEM_I2C_EXT0_CLK_EN;
-    case PERIPH_I2S1_MODULE:
-        return SYSTEM_I2S1_CLK_EN;
+    case PERIPH_I2S0_MODULE:
```

Comment:
Although esp32c3 has a single I2S channel, esp-idf keeps its name referenced as "1".  
I suggest not changing it (and keep as is `I2S1`). PTAL: 
https://github.com/espressif/esp-idf/blob/master/components/hal/esp32c3/include/hal/clk_gate_ll.h#L38C10-L39C35  
https://github.com/espressif/esp-idf/blob/master/components/soc/esp32c3/include/soc/periph_defs.h

Reply:
why esp-idf keeps 1? This is confusing, when reading it, we expect I2S0 as well

As esp-idf keeps 1, I agree to keep 1 in Zephyr

Reply:
I pushed the change :heavy_check_mark: